### PR TITLE
[mempool] more/updated logging + counters

### PR DESCRIPTION
**Pull request by christinacelee**
## Motivation

(Write your motivation for proposed changes here.)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/libra/libra/blob/master/CONTRIBUTING.md#pull-requests)?

(Write your answer here.)

## Test Plan

(Share your test plan here. If you changed code, please provide us with clear instructions for verifying that your changes work.)

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/libra/libra/developers.libra.org, and link to your PR here.)

### CODE REVIEW

File: mempool/src/shared_mempool/tasks.rs [hunk 1]
```diff
@@ -104,18 +106,23 @@ pub(crate) async fn process_transaction_broadcast<V>(
     request_id: Vec<u8>,
     timeline_state: TimelineState,
     peer: PeerNetworkId,
+    timer: HistogramTimer,
 ) where
     V: TransactionValidation,
 {
+    timer.stop_and_record();
```
Comment (bmwill) [inline mempool/src/shared_mempool/tasks.rs:113]: We we actually want to stop the time right when entering the function? I thought these timers were for execution time or is that captured by the timer below?

**Review by bmwill — APPROVED** [review]
Only one question before you land this

File: mempool/src/shared_mempool/tasks.rs (hunk 1 above)
Comment (christinacelee) [reply mempool/src/shared_mempool/tasks.rs:113]: yes, so the timer passed in would measure the time b/w this task being scheduled by the bounded executor and actually started. the timer below is how the latency of this started task itself

**Pull request closed without merging**